packages/lit-dev-content/samples/examples/async-task/npm-info.ts

```diff
@@ -1,57 +1,73 @@
 import {LitElement, css, html} from 'lit';
 import {customElement, state} from 'lit/decorators.js';
-import {Task} from '@lit-labs/task';
+import {map} from 'lit/directives/map.js';
+import {Task, initialState} from '@lit/task';
 import {fetchPackageInfo} from './npm.js';
 
 @customElement('npm-info')
 export class NpmInfo extends LitElement {
   @state()
-  private _package = 'lit';
+  private _packageName = 'lit';
 
-  private _fetchNpmInfoTask = new Task(this, {
+  /*
+   * This is the async Task that fetches data from npm.
+   *
+   * The current state of the task is rendered below with
+   * `this._npmInfoTask.render()`.
+   */
+  private _npmInfoTask = new Task(this, {
     task: async ([pkgName], {signal}) => {
       if (pkgName === undefined || pkgName === '') {
-        throw new Error('Empty package name');
+        // This puts the task back into the INITIAL state
+        return initialState;
       }
       return await fetchPackageInfo(pkgName, signal);
```

Comment:
To leave a record here, @rictic had a nice suggestion to do a debounce here by adding
```ts
// debounce
await new Promise((r) => setTimeout(r, 50));
signa.throwIfAbort();
```
Then the task can be triggered by calling on `input` event rather than `change` event.